parsec/backend/postgresql/user_queries/find.py

```diff
@@ -1,14 +1,28 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
-
 from pendulum import now as pendulum_now
 from functools import lru_cache
-from typing import Tuple, List
+from typing import Tuple, List, Optional
 
 from parsec.api.protocol import UserID, OrganizationID, HumanHandle
 from parsec.backend.user import HumanFindResultItem
 from parsec.backend.postgresql.utils import query
 from parsec.backend.postgresql.queries import Q, q_organization_internal_id
 
+_q_retrieve_active_human_by_email = Q(
+    f"""
+SELECT
+    user_.user_id,
+    human.email,
+    human.label,
+    user_.revoked_on IS NOT NULL AND user_.revoked_on <= $now as revoked
```

Comment:
Given we return the "active" human if any, we should move this line on the filder part of the query and modify it to be:
```sql
user_.revoked_on IS NULL OR user_.revoked_on > $now
```